src/install/lockfile/Package/Meta.zig

```diff
@@ -9,7 +9,8 @@ pub const Meta = extern struct {
 
     arch: Npm.Architecture = .all,
     os: Npm.OperatingSystem = .all,
-    _padding_os: u16 = 0,
+    libc: Npm.Libc = .all,
```

Comment:
i think we want `Libc.all` to equal `0` if it's the default value here. reason for this is old binary lockfiles would set these 8 bits to zero and ignore them. now that we read them we want to see them as the default rather than `Libc.none`